### PR TITLE
add const

### PR DESCRIPTION
**Pull request by leonbiton1**

### CODE REVIEW

**Review by offerm — COMMENTED** [review]
you should be use these too, no?

Is it good to be bool?
maybe enum is better?

File: upscale.go [hunk 1]
```diff
@@ -315,9 +322,9 @@ func AddTransactionEvents(source string, isTx bool, count int32) {
 	}
 	copy(sourceID[:], id)
 	if activePeer, ok := self.activePeers[sourceID]; ok {
-		if isTx {
+		if txType == TX {
```
Comment (offerm) [inline upscale.go:325]: please change to switch.
in default issue an error message.

File: upscale.go [hunk 2]
```diff
@@ -315,10 +322,14 @@ func AddTransactionEvents(source string, isTx bool, count int32) {
...
 	if activePeer, ok := self.activePeers[sourceID]; ok {
-		if isTx {
+		switch txType {
+		case TX:
 			activePeer.TxsSent += count
-		} else {
+		case ANNOUNCEMENT:
 			activePeer.NotificationSent += count
+		default:
+			log.Errorf("error reciveing type %v, type should bt tx or announcement", txType)
```
Comment (offerm) [inline upscale.go:331]: fix type please

File: upscale.go [hunk 3]
```diff
@@ -17,6 +17,13 @@ import (
 	"time"
 )
 
+type TransactionType int
```
Comment (offerm) [inline upscale.go:20]: bad name

Better name TransactionEvent

Question - should this be exported?

Comment (leonbiton1) [reply upscale.go]: fixed and yes we should use it in gw

**Review by offerm — APPROVED** [review]

**Pull request merged**